seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -3912,7 +3912,7 @@
         "<blockquote>var myObj = {<br>  \"Space Name\": \"Kirk\",<br>  \"More Space\": \"Spock\"<br>};<br>myObj[\"Space Name\"]; // Kirk<br>myObj['More Space']; // Spock</blockquote>",
         "Note that property names with spaces in them must be in quotes (single or double).",
         "<h4>Instructions</h4>",
-        "Read the values of the properties <code>\"an entree\"</code> and <code>\"the drink\"</code> of <code>testObj</code> using bracket notation."
+        "Read the values of the properties <code>\"an entree\"</code> and <code>\"the drink\"</code> of <code>testObj</code> using bracket notation and assign them to <code>var entreeValue</code> and <code>var drinkValue</code> respectively."
```

Comment:
Please remove the `var` in front of the variables (e.g. change `var entreeValue` to just `entreeValue`).